compiler/apps/playground/__tests__/e2e/page.spec.ts

```diff
@@ -142,7 +160,7 @@ test('reset button works', async ({page}) => {
 });
 
 TEST_CASE_INPUTS.forEach((t, idx) =>
-  test(`playground compiles: ${t.name}`, async ({page}) => {
+  test.only(`playground compiles: ${t.name}`, async ({page}) => {
```

Comment:
```suggestion
  test(`playground compiles: ${t.name}`, async ({page}) => {
```

Reply:
Thanks for the catch!!
I have a bad habit of procrastinating by writing QOL commits, but forgetting to submit them as a PR 🤦‍♀️ This is actually from ~2 weeks ago